From da86e16a54874bcf8392c28e3877a4a75a8937c8 Mon Sep 17 00:00:00 2001
From: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Date: Fri, 2 Dec 2022 17:18:27 +0100
Subject: [PATCH] arm64: defconfig: enable rpmsg-i2c in addons fragment

Add the support of the I2C adapter over rpmsg.

Signed-off-by: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Change-Id: If9c751eaea98245854fd32f085d5aa749df304f4
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/310428
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Domain-Review: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -2,6 +2,7 @@
 CONFIG_CLKSRC_STM32_LP=y
 CONFIG_COUNTER=m
 CONFIG_DYNAMIC_DEBUG=y
+CONFIG_I2C_RPMSG=m
 CONFIG_I2C_STM32F7=y
 CONFIG_IIO_STM32_LPTIMER_TRIGGER=m
 CONFIG_IIO_STM32_TIMER_TRIGGER=m
